twindle-cli/package.json

```diff
@@ -32,7 +33,7 @@
     "puppeteer": "^5.4.1",
     "smtp-client": "^0.3.3",
     "string.prototype.matchall": "^4.0.2",
-    "yargs": "^16.1.0",
-    "twemoji": "13.0.1"
+    "twemoji": "13.0.1",
```

Comment:
why? 😊

Reply:
Twemoji?

Reply:
You just changed the order?

There is no need for any package json changes

Reply:
I didn't change order. It just happened.

Maybe directly doing `npm i package` reorders as alphabetical.

Reply:
Remove package json related changes